content/blog/2016-09-28-interview-with-incopro.md

```diff
@@ -0,0 +1,94 @@
+---
+title: Interview with Incopro
+created_at: 2016-09-28
+kind: article
+author_name: Brian Brazil
+---
+
+*Continuing our series of interviews with users of Prometheus, INCOPRO talks
+about how they went from Zabbix and Monit to Prometheus.*
+
+## Can you tell us about yourself and what Incorpo does?
+
+[INCOPRO](http://www.incopro.co.uk/) is a brand protection company. It uses
+technology, combined with IP and investigation expertise, to deliver effective
+and targeted protection to businesses looking to protect their IP assets
+online. Its Talisman brand protection system is used by many of the largest
+global brands to track and assess infringement issues across Marketplaces,
+Websites, Social Media, App Stores and more. INCOPRO’s expert analyst team use
+the Talisman technology and a range of language skills (including Japanese,
+Korean, Arabic, Russian, Mandarin, Cantonese, Punjabi etc.) to deliver
+prioritised enforcement against commercial scale infringements. INCOPRO was
+founded by a law firm and operates from its own central London HQ. The
+company's use of technology to deliver scalable enforcement was recognised by
+the Financial Times newspaper as the most innovative development in IP in 2015.
+
+My name is Akos Veres, Operations Engineer at INCOPRO, I had the task of
+setting up the proper monitoring solution for our company, as well as help with
+day to day operations in the short term. I became the only Ops at the company
+so everything falls under my jurisdiction.
+
+
+## What was your pre-Prometheus monitoring experience?
+
+There wasn’t a real solution, especially no alerting in place.
+[Zabbix](http://www.zabbix.com/) and [Monit](https://mmonit.com/) have been
+used to some extent, when things started to get out of control as the company
+grew bigger a new solution was needed and Prometheus was found back in December
```

Comment:
There are some commas or other things missing here.

How about something like: "control. As the company grew bigger, a new solution was needed..."